src/test/fuzz/package_eval.cpp

```diff
@@ -307,7 +307,7 @@ FUZZ_TARGET(tx_package_eval, .init = initialize_tx_pool)
             // just use result_package.m_state here. This makes the expect_valid check meaningless, but
             // we can still verify that the contents of m_tx_results are consistent with m_state.
             const bool expect_valid{result_package.m_state.IsValid()};
-            Assert(!CheckPackageMempoolAcceptResult(txs, result_package, expect_valid, nullptr));
+            Assert(!CheckPackageMempoolAcceptResult(txs, result_package, expect_valid, &tx_pool));
```

Comment:
5dd74f9c6ddc218417bda841f493ca2ed3f2b327

I don't 100% remember why we did this, but I think it was so we don't check mempool contents when `test_accept=true`?

Reply:
This should only be called when not `test_accept`, right?

Reply:
I think so...

Reply:
Aha yes. Can resolve.